testsuite/objects/__init__.py

```diff
@@ -104,9 +121,9 @@ class Property:
     name: str
     value: ABCValue
 
-    def asdict(self):
+    def asdict_override(self):
         """Override `asdict` function"""
-        return {"name": self.name, **asdict(self.value)}
+        return asdict_compose(self)
```

Comment:
To be honest, I dont think this is an improvement as the override function is still there